lambdas/backend-api/README.md

```diff
@@ -170,3 +170,32 @@ curl -X POST --location "${APIG_STAGE}/v1/routing-configuration" \
   "cascadeGroupOverrides": [{ "name": "standard" }],
   "name": "RC name"
 }'
+```
+
+### PUT - /v1/routing-configuration/:routingConfigId - Update a routing configuration
```

Comment:
I think I forgot to add one of these for the count endpoint, would you mind adding one for that?